functions/src/import/config.ts

```diff
@@ -0,0 +1,204 @@
+    'Number of documents to stream into Meilisearch at once.',
+    value => parseInt(value, 10),
+    300
+  )
+  .option(
+    '-H, --host <host>',
+    'The Host of your meilisearch database. Example: http://localhost:7700.'
+  )
+  .option(
+    '-a, --api-key <api-key>',
+    'The Meilisearch API key with permission to perform actions on indexes. Both the private key and the master key are valid choices but we strongly recommend using the private key for security purposes.'
+  )
+
+const validateInput = (
+  value: string,
+  name: string,
+  regex: RegExp,
+  sizeLimit: number
+) => {
+  if (!value || value === '' || value.trim() === '') {
+    return `Please supply a ${name}`
+  }
+  if (value.length >= sizeLimit) {
+    return `${name} must be at most ${sizeLimit} characters long`
+  }
+  if (!value.match(regex)) {
+    return `The ${name} does not match the regular expression provided`
+  }
+  return true
+}
+
+const validateBatchSize = (value: string) => {
+  return parseInt(value, 10) > 0
+}
+
+// Questions for the interactive user interface in command line.
+const questions = [
+  {
+    message: 'What is your Firebase project ID?',
+    name: 'project',
+    type: 'input',
+    validate: value =>
+      validateInput(
+        value,
+        'project ID',
+        FIRESTORE_VALID_CHARACTERS,
+        PROJECT_ID_MAX_CHARS
+      ),
+  },
+  {
+    message:
+      'What is the path of the Cloud Firestore Collection you would like to import from? ' +
+      '(This may, or may not, be the same Collection for which you plan to mirror changes.)',
+    name: 'sourceCollectionPath',
+    type: 'input',
+    validate: value =>
+      validateInput(
+        value,
+        'collection path',
+        FIRESTORE_VALID_CHARACTERS,
+        FIRESTORE_COLLECTION_NAME_MAX_CHARS
+      ),
+  },
+  {
+    message: 'Would you like to import documents via a Collection Group query?',
+    name: 'queryCollectionGroup',
+    type: 'confirm',
+    default: false,
+  },
+  {
+    message:
+      "What is the Uid of the Meilisearch index that you would like to use? (A index will be created if it doesn't already exist)",
+    name: 'index',
+    type: 'input',
+    validate: value =>
+      validateInput(
+        value,
+        'index',
+        MEILISEARCH_VALID_CHARACTERS,
+        MEILISEARCH_UID_MAX_CHARS
+      ),
+  },
+  {
+    message:
+      'How many documents should the import stream into Meilisearch at once?',
+    name: 'batchSize',
+    type: 'input',
+    default: 300,
+    validate: validateBatchSize,
+  },
+  {
+    message:
+      'What is the host of the Meilisearch database that you would like to use? Example: http://localhost:7700.',
+    name: 'host',
+    type: 'input',
+  },
+  {
+    message:
+      'What The Meilisearch API key with permission to perform actions on indexes. Both the private key and the master key are valid choices but we strongly recommend using the private key for security purposes.',
```

Comment:
I think there are some words missing here : 
`What The Meilisearch API key with permission to perform actions on indexes`